scripts/update.js

```diff
@@ -328,6 +325,10 @@ const extractDomain = (repoUrl) => {
 }
 
 const updateRepoStats = (url, delta) => {
+  if (typeof url !== 'string') {
+    return
+  }
```

Comment:
Are you sure this will keep the count correctly over time? A package that was `+1` for having a url but is now null should be `-1`.

Reply:
Yes, this is correct. The logic you described is handled in the `apply` function. This check is needed to simplify the code there and to avoid bugs if a "bad" URL was passed as an argument.
I would like to add the tests for it but it will require significant changes to the `update.js`. 
Also, @LucasScotta is working on tests for stats so probably I will wait for his changes to avoid conflicts.